src/ducks/lagret-filter-ui-state.ts

```diff
@@ -0,0 +1,111 @@
+import {OrNothing} from "../utils/types/types";
+import {LagretFilter} from "./lagretFilter";
+
+// Actions
+export const VELG_MINE_FILTER = 'lagretfilter_velg/VELG_MINE_FILTER';
+export const MARKER_MINE_FILTER = 'lagretfilter_velg/MARKER_MINE_FILTER';
+export const AVMARKER_MINE_FILTER = 'lagretfilter_velg/AVMARKER_MINE_FILTER';
+export const AVMARKER_SISTE_VALGT_MINE_FILTER = 'lagretfilter_velg/AVMARKER_SISTE_VALGT_MINE_FILTER';
+
+export const MARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/MARKER_VEILEDER_GRUPPE';
+export const AVMARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/AVMARKER_VEILEDER_GRUPPE';
+
+export const APEN_MINE_FILTER_MODAL = 'lagretfilter_velg/APEN_MINE_FILTER_MODAL';
+export const LUKK_MINE_FILTER_MODAL = 'lagretfilter_velg/LUKK_MINE_FILTER_MODAL';
+
+
+export interface LagretFilterUIState {
+    valgtLagretFilter: OrNothing<LagretFilter>;
+    valgtVeilederGruppe: OrNothing<LagretFilter>;
+    sisteValgtLagretFilter: OrNothing<number>
```

Comment:
should this be sisteValgtMineFilter?

Reply:
The same as for previous...its used in veileder grupper as well..

Reply:
but the line above says `valgtVeiledergrupper`, so this is `sisteValgtMineFilter`